src/Chapter2.hs

```diff
@@ -621,7 +638,10 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = [] 
+takeEven (x:_:xs) = [x] ++ takeEven xs
```

Comment:
Wonderful, great pattern-matching skills 💯 

It's better to use `:` here as well:

```suggestion
takeEven (x:_:xs) = x : takeEven xs
```